src/main/java/io/github/mzmine/modules/dataprocessing/id_lipididentification/lipids/TableModel.java

```diff
@@ -0,0 +1,133 @@
+package io.github.mzmine.modules.dataprocessing.id_lipididentification.lipids;
+
+public class TableModel {
+
+  private String id;
+  private String lipidCoreClass;
+  private String lipidMainClass;
+  private String lipidClass;
+  private String molecularFormula;
+  private String abbreviation;
+  private String ionization;
+  private String exactMass;
+  private String info;
+  private String status;
+  private String msmsFragmentsPos;
+  private String msmsFragmentsNeg;
+
+  public TableModel(String id, String lipidCoreClass, String lipidMainClass, String lipidClass,
```

Comment:
I think the "TableModel" name is too generic. Maybe this class could be named "LipidClassDescription" or something like that?